components/Layout/SectionWrapper.jsx

```diff
@@ -21,4 +24,16 @@ const SectionWrapper = ({ children, customClasses = "px-8 py-12 lg:p-24", backgr
   );
 };
 
+SectionWrapper.propTypes = {
+  backgroundType: PropTypes.string,
+  children: PropTypes.element.isRequired,
+  customClasses: PropTypes.string,
+  id: PropTypes.string,
+};
+SectionWrapper.defaultProps = {
+  backgroundType: null,
+  customClasses: null,
```

Comment:
Maybe this is where the padding issue is coming from?

Reply:
Fixed, used cn in className, also provided correct customClasses, somehow missed that